source/core.py

```diff
@@ -424,7 +424,12 @@ def main():
 	"""
 	log.debug("Core starting")
 
-	ctypes.windll.user32.SetProcessDPIAware()
+	if winVersion.getWinVer() >= winVersion.WIN10_1703:
+		ctypes.windll.user32.SetProcessDpiAwarenessContext(-4)  # DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2
+	elif winVersion.getWinVer() >= winVersion.WIN81:
+		ctypes.windll.shcore.SetProcessDpiAwareness(2)  # PROCESS_PER_MONITOR_DPI_AWARE
+	else:
+		ctypes.windll.user32.SetProcessDPIAware()

```

Comment:
As mentioned in the PR discussion it may be worth to set DPI awareness via API calls only for source copies. Also while at it it would be nice to check the results returned from these functions and log a warning if they fail.

Reply:
I hope to merge https://github.com/nvaccess/nvda/pull/14015 before implementing this change